pkg/cover/backend/dwarf.go

```diff
@@ -261,8 +261,9 @@ func makeDWARFUnsafe(params *dwarfParams) (*Impl, error) {
 		Symbolize: func(pcs map[*Module][]uint64) ([]Frame, error) {
 			return symbolize(target, objDir, srcDir, buildDir, splitBuildDelimiters, pcs)
 		},
-		RestorePC:   makeRestorePC(params, pcBase),
-		CoverPoints: allCoverPointsMap,
+		RestorePC:              makeRestorePC(params, pcBase),
+		CallbackPoints:         allCoverPointsMap,
```

Comment:
Don't we need corresponding changes for other backends?

Reply:
Added "if rg.CoverageCallbackPoints != nil" to DoCoverJSONL() to not crash on other platforms.
gVisor already do full symbolization.
Other backends are not used on syzbot afais.

/cover?jsonl=1 returns information about all coverage points for gVisor and DWARF.
/cover?jsonl=1 returns information about only reached by fuzzer files for Mach O.
Added these details as a function comment.

Does it look enough?

Reply:
https://github.com/google/syzkaller/issues/4585#issuecomment-2007874590 will simplify this part later.
DWARF will work the same way as gVisor now (symbolize all on init).